README.md

```diff
@@ -28,6 +28,7 @@ Check out the Kubebuilder [book](https://book.kubebuilder.io).
 
 - Kubebuilder Book: [book.kubebuilder.io](https://book.kubebuilder.io)
 - GitHub Repo: [kubernetes-sigs/kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+- Tutorial: [CronJob tutorial source code](https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/cronjob-tutorial/testdata/project)
```

Comment:
I am not sure if we should add it because the master branch not necessary will have the same version of the doc. 
See that the docs in production are from another branch.

Reply:
Perhaps it would be more useful to have the docs point to the example code itself, if they don't already? And modify the label above to read `Kubebuilder Book and Tutorial` so folks know they can find the tutorial within the book?

Reply:
Hi @gabbifish, 

Currently the testdata is outdated and it has not been updating to match with the version/doc. So, it might bring confusion. Also, we cannot point to the master branch. We would need to work with the tags releases instead of it.